createImaPluginConfig.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable jsdoc/require-jsdoc */
```

Comment:
Maybe these default configurations could be part of the `ima-plugin-cli`, since we will be re-using them in different repositories

Reply:
Yup totally agree, I would probably include them with the migration to `@ima` monorepo if it's ok?

Reply:
Fixed with new plugin version